operations/mimir-mixin/docs/playbooks.md

```diff
@@ -213,6 +217,11 @@ How to **investigate**:
         - Check `Memcached Overview` dashboard
         - If memcached eviction rate is high, then you should scale up memcached replicas. Check the recommendations by `Mimir / Scaling` dashboard and make reasonable adjustments as necessary.
         - If memcached eviction rate is zero or very low, then it may be caused by "first time" queries
+      - Cache query timeouts
+        - Check store gateway logs and look for warnings about timed out Memcached queries
```

Comment:
Please spell it "store-gateway"

Reply:
Can you show an example of log message to grep?

Reply:
@pracucci I added an example similar to what I used myself. It's pretty ad-hoc, but does the job.